independent-publisher-connectors/Safaricom Mpesa Daraja/apiDefinition.swagger.json

```diff
@@ -0,0 +1,620 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "daraja",
+    "description": "A connector that exposes daraja mpesa endpoints",
+    "version": "1.0",
+    "contact": 
+    {
+      "name": "John Muchiri, Bernard Karaba",
+      "url": "https://johnmuchirim.github.io/",
+      "email": "jeanfrancaiseharicot@gmail.com",
+      "email": "bkaraba14@gmail.com"
+    }
+  },
+  "host": "sandbox.safaricom.co.ke",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+   "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/oauth/v1/generate": {},
+    "/mpesa/stkpush/v1/processrequest": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "MerchantRequestID": {
+                  "type": "string",
+                  "description": "MerchantRequestID"
+                },
+                "CheckoutRequestID": {
+                  "type": "string",
+                  "description": "CheckoutRequestID"
+                },
+                "ResponseCode": {
+                  "type": "string",
+                  "description": "ResponseCode"
+                },
+                "ResponseDescription": {
+                  "type": "string",
+                  "description": "ResponseDescription"
+                },
+                "CustomerMessage": {
+                  "type": "string",
+                  "description": "CustomerMessage"
+                }
+              }
+            }
+          }
+        },
+        "summary": "lipa na mpesa",
```

Comment:
@johnmuchiriM all information within the swagger file should be in English language.